src/refmterr/lib/dune

```diff
@@ -13,3 +13,4 @@
 (library (name Refmterr)
     (public_name refmterr.lib)
     (libraries atdgen re.pcre unix pastel.lib))
+(copy_files# ../../../common/*)
```

Comment:
Is this related to this diff?

Reply:
This was the same error you fixed separately. I thought it came out of a merge conflict.